src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -82,6 +83,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case FindCommand.COMMAND_WORD:
             return new FindCommandParser().parse(arguments);
 
+        case FindTaskCommand.COMMAND_WORD:
+            return new FindTaskCommandParser().parse(arguments);
```

Comment:
Might be better to add a test in `AddressBookParserTest` to ensure that the new command is being parsed correctly!